src/App.js

```diff
@@ -1,10 +1,19 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+// import Home from './components/Home';
+// import Search from './components/Search';
+// import { ThemeProvider } from 'styled-components';
```

Comment:
I'd use the `Home` and `Search` components and remove the `ThemeProvider` comment since you don't have a theme yet. It's best to have clean commits without lef-over comments 😉